tests/test_occurrences_client.py

```diff
@@ -1,8 +1,21 @@
 import datetime
 
-from geopandas import GeoDataFrame
-from pandas import DataFrame
-from pandas.testing import assert_frame_equal
+try:
+    from geopandas import GeoDataFrame
+
+    HAS_GEOPANDAS = True
+except ImportError:
+    HAS_GEOPANDAS = False
+try:
+    from pandas import DataFrame
+
+    HAS_PANDAS = True
+except ImportError:
+    HAS_PANDAS = False
+try:
```

Comment:
Readability (if autoformat allows)

```suggestion
    HAS_PANDAS = False

try:
```